core/workflow-core/src/main/scala/edu/uci/ics/amber/core/storage/DocumentFactory.scala

```diff
@@ -24,6 +29,92 @@ object DocumentFactory {
 
   private def sanitizeURIPath(uri: URI): String = uri.getPath.stripPrefix("/").replace("/", "_")
 
+  private def createIcebergDocument(
```

Comment:
Since this private util func is only called once, I think there is no need to make it a function